### PR TITLE
Adding nerval recipe

### PR DESCRIPTION
**Pull request by mdadda**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mdadda** [comment]
@conda-forge/help-python The recipe has passed all the checks and is ready for review. Thank you.

File: recipes/nerval/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - pandas
+    - numpy
+    - matplotlib
```
Comment (synapticarbors) [inline recipes/nerval/meta.yaml:24]: Can you just confirm that you indeed need the full `matplotlib` package that includes Qt and not just the `matplotlib-base`?

**Comment by mdadda** [comment]
Hi,  matplotlib-base is enough. Shall I resubmit or are you going to make
the change?
Thanks.

**Comment by synapticarbors** [comment]
@mdadda -- Just push a new commit to your fork on this branch.

**Pull request merged**